Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -325,8 +297,6 @@ - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)
 
 - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)isReferrable explicitlyRequestedReferrable:(BOOL)explicitlyRequestedReferrable automaticallyDisplayController:(BOOL)automaticallyDisplayController {
     self.shouldAutomaticallyDeepLink = automaticallyDisplayController;
-    self.preferenceHelper.isReferrable = isReferrable;
```

Comment:
this was one of those changes I felt uncomfortable making as I think we had this method in the onboarding documentation for a long time.